src/components/AppBreadcrumbs/AppBreadcrumbs.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Breadcrumbs, Link } from '@material-ui/core';
+import { Link as RouterLink } from 'react-router-dom';
+
+function capitalize(str: string): string {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
```

Comment:
I think this should be a const function:
```suggestion
const capitalize(str: string) = (string) => ({
  return str.charAt(0).toUpperCase() + str.slice(1);
});
```